src/Microsoft.AspNetCore.Razor/Tokenizer/Internal/CSharpKeywordDetector.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Tokenizer.Symbols;
 
-namespace Microsoft.AspNetCore.Razor.Tokenizer
+namespace Microsoft.AspNetCore.Razor.Tokenizer.Internal
 {
     internal static class CSharpKeywordDetector
```

Comment:
Will unchange namespace, talked to Eilon and might as well just keep the internal guys internal and where they wer eoriginally